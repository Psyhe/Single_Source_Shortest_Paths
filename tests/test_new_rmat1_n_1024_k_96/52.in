1024 572 582
572 274 155
572 3 255
572 261 228
572 288 204
572 192 39
574 64 192
576 1 55
576 144 187
576 72 3
576 781 183
576 640 18
576 208 163
576 41 184
576 40 97
576 608 246
576 0 214
576 288 239
576 792 122
576 4 163
576 900 220
576 256 155
576 256 89
576 3 74
576 18 206
576 256 159
576 144 241
576 2 134
576 64 99
576 642 228
576 6 13
576 202 211
576 512 133
576 648 179
576 2 125
576 528 211
576 770 1
576 19 138
576 12 3
576 790 255
576 16 18
576 70 118
576 352 218
576 161 183
576 2 109
576 1 31
576 44 201
576 256 83
576 260 30
576 128 116
576 0 65
576 132 197
576 406 124
576 256 72
576 673 52
576 22 136
576 40 115
576 776 169
576 304 25
576 642 20
576 8 82
576 2 104
576 1 13
576 309 142
576 5 107
576 648 213
576 592 243
576 4 141
576 84 147
576 182 107
576 2 108
576 149 53
576 16 250
576 0 119
576 256 192
576 144 72
576 75 227
576 513 183
576 8 145
576 160 203
576 16 120
576 16 12
576 610 239
576 114 89
576 678 131
576 456 203
576 72 134
576 640 175
576 36 145
576 268 237
576 2 160
576 0 235
576 327 1
576 6 39
576 106 111
576 288 249
576 65 45
576 96 121
576 512 25
576 304 87
576 42 144
576 515 107
576 176 199
576 66 140
576 128 26
576 257 166
576 64 41
576 4 3
576 33 20
576 368 199
577 64 31
577 257 156
577 128 218
577 66 66
577 514 113
577 6 17
577 427 10
577 1 144
577 384 16
577 72 184
577 0 15
577 32 233
577 20 183
577 520 161
577 64 248
577 5 35
577 0 122
577 823 64
577 584 118
577 0 126
577 448 114
577 520 219
577 600 42
577 161 50
577 386 89
577 2 220
577 704 114
577 208 72
577 0 178
577 406 182
577 308 239
577 4 208
577 164 81
577 641 118
577 104 60
577 164 210
577 128 32
577 322 197
577 64 170
577 704 220
577 2 196
578 264 78
578 556 241
578 128 208
578 264 254
578 0 170
578 513 238
578 128 23
578 147 49
578 33 30
578 256 36
578 0 152
578 647 44
578 0 79
578 537 11
578 65 152
578 128 76
578 128 254
578 544 103
578 270 205
578 336 64
578 160 172
578 71 25
578 24 236
578 257 114
578 8 149
578 22 247
578 80 135
578 256 55
578 20 205
578 36 17
578 642 194
578 325 126
578 17 131
579 129 56
579 272 58
579 785 122
579 132 135
579 330 232
579 66 169
579 164 70
579 278 137
579 552 197
579 512 120
580 73 15
580 6 113
580 276 108
580 47 49
580 68 229
580 25 128
580 640 13
580 0 194
580 327 160
580 36 95
580 132 246
580 96 211
580 520 207
580 70 213
580 512 49
580 0 226
580 64 8
580 896 43
580 66 142
580 0 111
580 275 109
580 16 114
580 34 60
580 0 38
580 676 202
580 2 130
580 666 187
580 544 199
580 247 152
580 781 48
580 323 217
580 64 90
580 28 156
580 807 15
581 294 247
581 16 232
581 72 231
581 769 148
581 530 90
581 16 58
581 424 178
581 320 176
581 22 66
581 260 184
582 0 107
582 0 228
582 16 33
582 96 54
582 0 152
582 560 225
582 281 251
582 44 100
582 81 75
582 264 169
582 584 162
582 640 94
