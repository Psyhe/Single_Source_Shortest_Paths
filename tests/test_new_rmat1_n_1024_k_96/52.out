47
1e+18
192
1e+18
5
15
19
63
8
63
38
